command/build_test.go

```diff
@@ -1112,3 +1112,62 @@ func TestBuildCommand_ParseArgs(t *testing.T) {
 		})
 	}
 }
+
+// TestBuildCmd aims to test the build command, with output validation
+func TestBuildCmd(t *testing.T) {
+	tests := []struct {
+		name         string
+		args         []string
+		expectedCode int
+		outputCheck  func([]byte, []byte) error
+	}{
+		{
+			name: "hcl - no build block error",
+			args: []string{
+				testFixture("hcl", "no_build.pkr.hcl"),
+			},
+			expectedCode: 1,
+			outputCheck: func(out, err []byte) error {
+				strerr := string(err)
+				if !strings.Contains(strerr, "Error: Missing build block") {
+					return fmt.Errorf("expected 'Error: Missing build block' in output, did not find it")
+				}
+
+				nbErrs := strings.Count(strerr, "Error: ")
+				if nbErrs != 1 {
+					return fmt.Errorf("error: too many errors in stdout for build block")
```

Comment:
It is helpful to indicate what we expected to see. 
```suggestion
					return fmt.Errorf("error: too many errors in stderr for build block. expected 1 got %d", n)
```